src/documentCreator/sample-config.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
will this be more apt to be placed in the text/fixture folder?

Reply:
also, why is there another copy in the test/fixture folder that is in the documentCreator folder?